RELEASE.md

```diff
@@ -0,0 +1,3 @@
+Release type: patch
+
+Fix wrong stack frame being used in strawberry.lazy()
```

Comment:
``````suggestion
This release fixes an issue that prevented using strawberry.lazy with relative paths.

The following should work now:

```python
@strawberry.type
class TypeA:
    b: Annotated["TypeB", strawberry.lazy(".type_b")]
```
``````